client/src/components/Exercise/view.js

```diff
@@ -69,7 +67,7 @@ class PureCreateExercise extends React.Component {
                 skin="icon"
                 size="medium"
                 className={style.setDelete}
-                onClick={() => onDeleteSet(set.id)}
+                onClick={() => onDeleteSet(data.id, set.id)}
```

Comment:
I strongly believe that this is unneeded change.
In consumer (e.g. in `editNewExercise/view.js`) you can do like:
```
_onDeleteSet = (setId) => {
    const exerciseId = this.props.data.id;
    this.props.onDeleteSet(exerciseId, setId);
  };
```

Reply:
btw, you already do it this way here https://github.com/vincefliek/YourGym/pull/13/files#diff-4716bcdda4b5b25bbe01854f308d61904f244ae3d33ccdc789570d904f99df33R9